.github/workflows/packaging-tests.yml

```diff
@@ -1,9 +1,10 @@
-name: Packaging
+name: "Packaging"
 
 on:
   pull_request:
+  push:
```

Comment:
Is the `pull_request` on the above line still supposed to be here? We want this to only run after merging to main, correct?

Do you know what happens if this job fails? Do we all get an email?

Reply:
> Is the pull_request on the above line still supposed to be here? We want this to only run after merging to main, correct?

I was envisioning this as a ci test that doesn't block merge.

> Do you know what happens if this job fails? Do we all get an email?

Unsure

Reply:
> I was envisioning this as a ci test that doesn't block merge.

I'm not necessarily against the idea, but I think if it is part of our per-PR CI, it might be confusing for contributors to have failing jobs that they don't need to fix. I'm ok with it being a post-merge job as long as we get notified somehow (and I think we do via email).

If you want it per PR, then why is `push` here also?

Reply:
> it might be confusing for contributors to have failing jobs that they don't need to fix

I've contributed to projects that use a similar non-blocking structure (a better message on failure would be clearer, of course). I'm fine either way.

Reply:
> I'm fine either way.

Same, except for the other part of my comment about `push`. Is there a reason we want `push` here?